docs/source/cluster_management/index.rst

```diff
@@ -10,3 +10,7 @@ Cluster Management
    clis
    console
 
+You can add the Dropwizard bundle to **Dropwizard applications**. This will add startup commands to launch
+the AtlasDB :ref:`Console <console>` and :ref:`CLIs <clis>` such as ``sweep`` and ``timestamp``,
+which are needed to perform :ref:`live backups <backup-restore>`. **These commands will only work if
```

Comment:
I think the CLIs are used directly for backups and this seems to imply that you can't do live backups without the dropwizard bundle.